tox.ini

```diff
@@ -20,12 +20,12 @@ deps =
 
 [testenv:integration]
 deps =
-    git+https://github.com/juju/python-libjuju.git
-    lightkube
+    juju
     pytest
     pytest-operator
     selenium
     selenium-wire
+    lightkube<0.11
```

Comment:
Should we have a test-requirements.txt?

Reply:
good question.  We seem inconsistent on this - I think we sometimes have either.  Maybe we should bring this up as a general code style thing that we make a team rule for?